### PR TITLE
Add ability for multiple user accounts

### PR DESCRIPTION
**Pull request by gemmai95**
Add ability for admin user to add multiple user accounts. This allows user accounts for developers to be added dynamically rather than manually.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[SDX Console - Multiple User Accounts (5)](https://trello.com/c/jLPfBHwv/364-sdx-console-multiple-user-accounts-5)

**Review by tej99 — DISMISSED** [review]

File: console/forms.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from flask_wtf import Form
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired
+
+
+class NewUserForm(Form):
+    email = StringField('email', validators=[DataRequired()])
+    password = PasswordField('password', validators=[DataRequired()])
```
Comment (JamesGardiner) [inline console/forms.py]: Are there any IA password requirements we need to set here?

File: console/views.py [hunk 2]
```diff
@@ -15,7 +15,8 @@
 from sqlalchemy.exc import DataError, SQLAlchemyError
 from structlog import wrap_logger
 
-from console.database import db, SurveyResponse
+from console.database import db, SurveyResponse, create_dev_user
```
Comment (JamesGardiner) [inline console/views.py]: alphabetize these imports

File: console/views.py [hunk 3]
```diff
@@ -90,10 +91,10 @@ def decrypt():
             decrypted_data = decrypt_response.text
             error = ""
 
-        return render_template('decrypt.html', decrypted_data=decrypted_data, error=error)
+        return render_template('decrypt.html', decrypted_data=decrypted_data, error=error, user=flask_security.core.current_user.email)
```
Comment (JamesGardiner) [inline console/views.py]: break these over lines

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: console/settings.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
 SECRET_KEY = os.getenv('CONSOLE_SECRET_KEY', 'secretwords')
 SECURITY_PASSWORD_SALT = os.getenv('CONSOLE_PASSWORD_SALT', '1ab')
 CONSOLE_LOGIN_TIMEOUT = os.getenv('CONSOLE_LOGIN_TIMEOUT', 10)
+CONSOLE_INITIAL_ADMIN_PASSW = os.getenv('CONSOLE_INITIAL_ADMIN_PASSW')
```
Comment (necrophonic) [inline console/settings.py]: That's a bit of an odd name for the var - no reason to truncate `PASSWORD` down to `PASSW`.

File: console/forms.py (hunk 1 above)
Comment (necrophonic) [reply console/forms.py]: I wouldn't go crazy (very much against too prescriptive password rules as they just make things less secure) but a minimum length wouldn't go amiss.

File: console/settings.py (hunk 4 above)
Comment (necrophonic) [reply console/settings.py]: Required environment envs should also be listed in the README file (see https://github.com/ONSdigital/sdx-store/blob/master/README.md)

File: console/templates/layout.html [hunk 5]
```diff
@@ -31,6 +31,11 @@
                 <a href="{{ href|e }}">{{ caption|e }}</a>
               </li>
               {% endfor %}
+              {% if current_user.email=="admin" %}
```
Comment (necrophonic) [inline console/templates/layout.html]: Wouldn't just check that the user's email is "admin" as that's probably pretty easy to spoof and elevate yourself to admin permissions. Should be checking permissions on the actual user from the db (admin user should be flagged differently)

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**